apps/explorer/src/ui/DisclosureBox.tsx

```diff
@@ -2,38 +2,68 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Disclosure } from '@headlessui/react';
-
-import { ReactComponent as ChevronDownIcon } from './icons/chevron_down.svg';
+import { ChevronDown24 } from '@mysten/icons';
+import { cva } from 'class-variance-authority';
 
 import type { ReactNode } from 'react';
 
 export type DisclosureBoxProps = {
     defaultOpen?: boolean;
     title: ReactNode;
     children: ReactNode;
+    variant: 'inline' | 'accordion';
 };
 
+const disclosureStyles = cva('', {
```

Comment:
was torn between just creating another disclosure component here to support the inline variant since we don't really have a standard for this in designs afaik